conf/application.conf

```diff
@@ -113,15 +112,6 @@ bannerNotice {
   }
 }
 
-
-loginPageNotice {
-  enabled = false
-  englishTitle = ""
-  englishBody = ""
-  welshTitle = ""
-  welshBody = ""
-}
-
```

Comment:
This piece of config should remain so we can add a banner to the top of home page.